local/README.md

```diff
@@ -0,0 +1,44 @@
+# local development scripts
+A home for experimental scripts used for launching Nextflow Tower workflows
+
+## Setup
+
+These instructions assume that you already have Python, a Python version manager (`pyenv`), and `pipenv` installed.
+
+### Python Environment
+
+Set up your Python environment for using these scripts by ruinning
+```
+pipenv install --dev
+pipenv shell
+```
+
+### Environment Variables
+
+In order for the scripts leveraging `py-orca` to connect to Nextflow Tower, you will need to configure the following environment variables:
+    - `NEXTFLOWTOWER_CONNECTION_URI`
+    - `SYNAPSE_CONNECTION_URI`
+    - `AWS_ACCESS_KEY_ID`
+    - `AWS_SECRET_ACCESS_KEY`
+    - `AWS_SESSION_TOKEN`
+    * Optionally, you can use `aws sso login` in place of the AWS environment variable credentials if you have pre-configured AWS profiles already.
+
+You can copy `local/.env.example` into a local `local/.env` file, replace the placeholders with your credentials, and run `source local/.env` in your terminal.
+
+## LENS
+
+Steps to run LENS workflow:
+1. Create necessary input files and upoload them to Synapse.
+    - The metaflow DAG requires 2 pre-formatted input files
+        - an input CSV file containing the LENS manifest information along with the Synapse URI's needed for `nf-synstage`
+    - A dataset YAML file containing a dataset ID and the Synapsee ID for the above described input CSV file
```

Comment:
```suggestion
        - an input CSV file containing the LENS manifest information along with the Synapse URI's needed for `nf-synstage`
        - A dataset YAML file containing a dataset ID and the Synapse ID for the above described input CSV file
```

Reply:
Nit: you might want to provide an example of a Synapse URI to show how it differs from just a Synapse ID.